examples/skeleton_detection/src/helperFunc.js

```diff
@@ -30,8 +30,8 @@ function getInput(inputElement) {
 
 function getURL(version) {
   let address;
-  const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
-  // const urlBase = '../skeleton_detection/model/';
+  // const urlBase = 'https://storage.googleapis.com/tfjs-models/weights/posenet/';
+  const urlBase = '../skeleton_detection/model/';
```

Comment:
it seems that we miss the model json & weights files

Reply:
We are unable to commit/push these files to this repo which may have license issue. 

Thanks @pinzhenx for the review BTW, :)

Reply:
If catch the model from local, do we need to add a README.md about how to get posenet model?

Reply:
I still have no good idea for users to get model if Google removed model json & weights files by json + bin and we don't have plan to update the new file support accordingly. Provide orginal model json & weights files in this repo may have license issue.

Reply:
@Wenzhao-Xiang Will add link https://drive.google.com/open?id=1UcNbGZyIS8tt-OwhwiP6o1PxbK4V5j1i  in readme this noon.

Reply:
@Wenzhao-Xiang The README have been updated via https://github.com/intel/webml-polyfill/pull/879/commits/2a1bf7b26829c9fdc381823eea6e2abbcb82772e , PTAL

Reply:
LGTM.